README.md

```diff
@@ -59,8 +59,10 @@ There is also a chat available via gitter for discussions:
 
 Not working, neither the internal speakers/microphone nor the headphone jack.
 
-What's working is audio via HDMI or any USB-connected audio device, so that can
-act as a workaround until internal audio is working.
+Alternatively, you can use:
+* audio via HDMI
+* bluetooth audio (see below)
```

Comment:
As A2DP doesn't work properly when using Bluetooth I wouldn't suggest using Bluetooth for audio.

Reply:
A2DP always did work for me (except on the latest kernels). I considered it a valid alternative because it does work for some; the bluetooth section is there to clarify if/when it works…
But if you prefer I'll remove that line.

Reply:
It'd be interesting to figure out under which conditions A2DP properly works. Until then I suggest removing it from the listed alternatives.